keyboards/xd96/info.json

```diff
@@ -7,11 +7,231 @@
+        {"label":"F5", "x":5, "y":0},
+        {"label":"F6", "x":6, "y":0},
+        {"label":"F7", "x":7, "y":0},
+        {"label":"F8", "x":8, "y":0},
+        {"label":"F9", "x":9, "y":0},
+        {"label":"F10", "x":10, "y":0},
+        {"label":"F11", "x":11, "y":0},
+        {"label":"F12", "x":12, "y":0},
+        {"label":"Insert", "x":13, "y":0},
+        {"label":"Delete", "x":14, "y":0},
+        {"label":"Home", "x":15, "y":0},
+        {"label":"End", "x":16, "y":0},
+        {"label":"PgUp", "x":17, "y":0},
+        {"label":"PgDn", "x":18, "y":0},
+
+        {"label":"~", "x":0, "y":1},
+        {"label":"!", "x":1, "y":1},
+        {"label":"@", "x":2, "y":1},
+        {"label":"#", "x":3, "y":1},
+        {"label":"$", "x":4, "y":1},
+        {"label":"%", "x":5, "y":1},
+        {"label":"^", "x":6, "y":1},
+        {"label":"&", "x":7, "y":1},
+        {"label":"*", "x":8, "y":1},
+        {"label":"(", "x":9, "y":1},
+        {"label":")", "x":10, "y":1},
+        {"label":"_", "x":11, "y":1},
+        {"label":"+", "x":12, "y":1},
+        {"label":"Backspace", "x":13, "y":1, "w":2},
+
+        {"label":"Num Lock", "x":15, "y":1},
+        {"label":"/", "x":16, "y":1},
+        {"label":"*", "x":17, "y":1},
+        {"label":"-", "x":18, "y":1},
+
+        {"label":"Tab", "x":0, "y":2, "w":1.5},
+        {"label":"Q", "x":1.5, "y":2},
+        {"label":"W", "x":2.5, "y":2},
+        {"label":"E", "x":3.5, "y":2},
+        {"label":"R", "x":4.5, "y":2},
+        {"label":"T", "x":5.5, "y":2},
+        {"label":"Y", "x":6.5, "y":2},
+        {"label":"U", "x":7.5, "y":2},
+        {"label":"I", "x":8.5, "y":2},
+        {"label":"O", "x":9.5, "y":2},
+        {"label":"P", "x":10.5, "y":2},
+        {"label":"{", "x":11.5, "y":2},
+        {"label":"}", "x":12.5, "y":2},
+        {"label":"|", "x":13.5, "y":2, "w":1.5},
+        {"label":"7", "x":15, "y":2},
+        {"label":"8", "x":16, "y":2},
+        {"label":"9", "x":17, "y":2},
+
+        {"label":"Caps Lock", "x":0, "y":3, "w":1.75},
+        {"label":"A", "x":1.75, "y":3},
+        {"label":"S", "x":2.75, "y":3},
+        {"label":"D", "x":3.75, "y":3},
+        {"label":"F", "x":4.75, "y":3},
+        {"label":"G", "x":5.75, "y":3},
+        {"label":"H", "x":6.75, "y":3},
+        {"label":"J", "x":7.75, "y":3},
+        {"label":"K", "x":8.75, "y":3},
+        {"label":"L", "x":9.75, "y":3},
+        {"label":":", "x":10.75, "y":3},
+        {"label":"\"", "x":11.75, "y":3},
+        {"label":"Enter", "x":12.75, "y":3, "w":2.25},
+
+        {"label":"4", "x":15, "y":3},
+        {"label":"5", "x":16, "y":3},
+        {"label":"6", "x":17, "y":3},
+        {"label":"+", "x":18, "y":2, "h":2},
+
+        {"label":"Shift", "x":0, "y":4, "w":2.25},
+        {"label":"Z", "x":2.25, "y":4},
+        {"label":"X", "x":3.25, "y":4},
+        {"label":"C", "x":4.25, "y":4},
+        {"label":"V", "x":5.25, "y":4},
+        {"label":"B", "x":6.25, "y":4}
```

Comment:
Lost a comma here.

```suggestion
        {"label":"B", "x":6.25, "y":4},
```

Reply:
That's what I get for doing this by hand...

Reply:
I have [quite a bit](https://github.com/qmk/qmk_firmware/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Anoroadsleft+in%3Atitle+Configurator) of practice at this. :wink: